test-cases/PR-provision-test.yaml

```diff
@@ -22,7 +22,7 @@ instance_provision: 'spot'
 
 gce_image_db: 'https://www.googleapis.com/compute/v1/projects/centos-cloud/global/images/family/centos-7'
 
-scylla_version: 4.4.1
+scylla_version: 4.6.3
```

Comment:
Don't you want to disable mgmt here too?

Reply:
No, we should cover management installation on those tests.

the reason I've disabled it for docker backend, is cause in docker backend we don't yes support manager installation. since there `real` monitor node. all command are using LocalCmdRunner (and if you try to install manager, it would be installed on your local machine, luckily for me, my OS isn't support by our manager tests, and it failed earlier)

Reply:
ok